examples/bundles/resnet50/main.cpp

```diff
@@ -17,6 +17,7 @@
 #include <png.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
```

Comment:
This isn't necessary is it?

Reply:
> This isn't necessary is it?

it is trying to fix compile error.

My environment is ubuntu 18.04, clang-6(which is from "apt install") and llvm-7(build from source code). 
my cmake command is:

> cmake .. -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug -DGLOW_WITH_BUNDLES=ON -DGLOW_WITH_CPU=ON -DCMAKE_PREFIX_PATH=/home/zhangwm/git/glow/llvm_install/

> make ResNet50Bundle

then, error message is:

> 

> /home/zhangwm/git/glow/examples/bundles/resnet50/main.cpp:252:52: error: use of undeclared identifier 'strlen'
>     if (!strncmp(config.symbolTable[i].name, name, strlen(name))) {
>                                                    ^
> /home/zhangwm/git/glow/examples/bundles/resnet50/main.cpp:281:3: error: use of undeclared identifier 'memset'; did you mean
>       'wmemset'?
>   memset(ptr, 0, size);
>   ^~~~~~
>   wmemset
> /usr/include/wchar.h:271:17: note: 'wmemset' declared here
> extern wchar_t *wmemset (wchar_t *__s, wchar_t __c, size_t __n) __THROW;

Reply:
Shouldn't memset be declared in `<cstring>`? See https://en.cppreference.com/w/cpp/string/byte/memset

Reply:
> Shouldn't memset be declared in `<cstring>`? See https://en.cppreference.com/w/cpp/string/byte/memset

yeah, seems csting is better than string.h in c++,  but it has same functions  with 
 string.h.

here is the cstring introduce page: https://en.cppreference.com/w/cpp/header/cstring

> This header was originally in the C standard library as <string.h>.
> This header is for C-style null-terminated byte strings.

 Since I am familiar c than c++, and there are already some head files like "stdlib.h" in main.cpp, then I used "string.h".

Should I recreate  PR  to modify to cstring, or this PR is just OK?

Reply:
@xizhizhang 

> Should I recreate PR to modify to cstring, or this PR is just OK?

No need to modify the PR. `cstring` is not any better than `string.h`

Reply:
> @xizhizhang
> 
> > Should I recreate PR to modify to cstring, or this PR is just OK?
> 
> No need to modify the PR. `cstring` is not any better than `string.h`

OK，thanks